test/app.test.js

```diff
@@ -64,4 +64,93 @@ describe('Main Generator', function () {
     });
   });
 
+  describe('Application generator without sample module 0.4.1', function () {
+    before(function (done) {
+      helpers.run(path.join(__dirname, '../app'))
+        .withOptions({
+          'skip-install': true
+        })
+        .withArguments([])
+        .withPrompts({
+          version: '0.4.1',
+          folder: 'temp',
+          appName: 'MEANtestAppName041',
+          appDescription: 'Testing the generator 041',
+          appKeywords: 'MongoDB, Express, AngularJS, Node.js',
+          appAuthor: 'Test',
+          addArticleExample: false,
+          addChatExample: false
+        })
+        .on('ready', function (generator) {
+          // this is called right before `generator.run()` is called
+        })
+        .on('end', function () {
+          done();
+        });
+    });
+
+    it('should generate a package.json file', function () {
+      assert.file('temp/package.json');
+    });
+
+    it('should have populated package.json correctly', function (done) {
+      fs.readFile('temp/package.json', function (err, data) {
+        if (err) {
+          done(err);
+        } else {
+          var packageJson = JSON.parse(data);
+          assert.equal(packageJson.name, 'meantestappname041');
+          assert.equal(packageJson.description, 'Testing the generator 041');
+          assert.equal(packageJson.author, 'Test');
+          done();
+        }
+      });
+    });
+  });
+
+  describe('Application generator without sample module 0.4.2', function () {
+    before(function (done) {
+      helpers.run(path.join(__dirname, '../app'))
+        .withOptions({
+          'skip-install': true
+        })
+        .withArguments([])
+        .withPrompts({
+          version: '0.4.2',
+          folder: 'temp',
+          appName: 'MEANtestAppName042',
+          appDescription: 'Testing the generator 042',
+          appKeywords: 'MongoDB, Express, AngularJS, Node.js',
+          appAuthor: 'Test',
+          addArticleExample: false,
+          addChatExample: false
+        })
+        .on('ready', function (generator) {
+          // this is called right before `generator.run()` is called
+        })
+        .on('end', function () {
+          done();
+        });
+    });
+
+    it('should generate a package.json file', function () {
+      assert.file('temp/package.json');
+    });
+
+    it('should have populated package.json correctly', function (done) {
+      fs.readFile('temp/package.json', function (err, data) {
+        if (err) {
+          done(err);
+        } else {
+          var packageJson = JSON.parse(data);
+          assert.equal(packageJson.name, 'meantestappname042');
+          assert.equal(packageJson.description, 'Testing the generator 042');
+          assert.equal(packageJson.author, 'Test');
+          assert.equal(packageJson['meanjs-version'], '0.4.2');
```

Comment:
Right, in that case can you just add a comment so that disparity is clear and not confusing in the future.